certified-connectors/Seismic Livedoc/apiDefinition.swagger.json

```diff
@@ -0,0 +1,614 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Seismic Livedoc",
+    "description": "Seismic Livedoc connector provides set of actions required for generating livedocs.",
+    "contact": {
+      "name": "Seismic",
+      "email": "support@seismic.com",
+      "url": "https://seismic.com"
+    },
+    "license": {
+      "name": "Seismic"
+    },
+    "version": "v2"
```

Comment:
@rkushvaha since this is a new connector, can you please change the version to "1.0"?

Reply:
added version 1.0